EcommerceApp/backend/spring-boot-ecommerce-app/src/main/resources/application.properties

```diff
@@ -5,18 +5,17 @@ spring.data.rest.base-path=/api
 
 #datasource
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/full-stack-ecommerce?useSSL=false&useUnicode=yes&characterEncoding=UTF-8&allowPublicKeyRetrieval=true&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/full-stack-ecommerce-liquibase
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+
 # db credentials
-#fixme add db credentials, removed from git(sensitive data)
+spring.datasource.username=githubtest
+spring.datasource.password=githubtest
+
 #timezone for hibernate timestamps
 spring.jpa.properties.hibernate.jdbc.time_zone=Europe/Moscow
-#restrict Hibernate table structure changes
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
It's not a good idea to use ddl-auto=update outside of test scope especially when we have liquibase to handle database migrations

Reply:
Done